rootex/core/language-locale.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "common/common.h"
+
+class Locale
+{
+	std::unordered_map<String, String> m_GameStrings;
+
+	Locale();
```

Comment:
Where is the language assigned during startup? How would you test if this class works?
Currently, this class is not instantiated at all. We better have someplace where this class is used, e.g. in Lua. This class will be most used in Lua and less inside the engine itself